doc/source/how-to-run-flower-with-deployment-engine.rst

```diff
@@ -0,0 +1,163 @@
+
+In this how-to guide, you will:
+
+- Set up a Flower project from scratch using the PyTorch template.
+- Run a Flower using the Deployment Engine.
+- Add a new federation configuration in the ``pyproject.toml`` file.
+- Start and monitor a run using the flwr CLI.
+
+The how-to guide should take less than 10 minutes to complete.
+
+Prerequisites
+-------------
+
+Before you start, make sure that:
+
+- The latest ``flwr`` CLI version is installed on your machine. Follow the installation
+  instructions :doc:`here <../how-to-install-flower>`.
+
+Step 1: Set Up
+--------------
+
+Create a new Flower project (PyTorch):
+
+.. code-block:: bash
+
+    $ flwr new my-project --framework PyTorch --username flower
+
+    🔨 Creating Flower project my-project...
+    🎊 Project creation successful.
+
+    Use the following command to run your project:
+
+          cd my-project
+          pip install -e .
+          flwr run
+
+    $ cd my-project
+    $ pip install -e .
+
+Step 2: Start the SuperLink
+---------------------------
+
+In a new terminal, start the SuperLink process in insecure mode:
+
+.. code-block:: bash
+
+    $ flower-superlink --insecure
+
+.. dropdown:: Understand the command
+
+    * ``flower-superlink``: Name of the SuperLink binary.
+    * | ``--insecure``: This flag tells the SuperLink to operate in an insecure mode, allowing
+      | unencrypted communication.
+
+Step 3: Start the SuperNodes
+----------------------------
+
+You will need two terminals for this step.
+
+1. **Terminal 1** Start the first SuperNode:
+
+   .. code-block:: bash
+
+       $ flower-supernode \
+            --insecure \
+            --superlink 127.0.0.1:9092 \
+            --node-config "partition-id=0 num-partitions=2" \
+            --supernode-address 127.0.0.1:9094 \
+            --isolation subprocess
+
+   .. dropdown:: Understand the command
+
+       * ``flower-supernode``: Name of the SuperNode binary.
+       * | ``--insecure``: This flag tells the SuperNode to operate in an insecure mode, allowing
+         | unencrypted communication.
+       * | ``--superlink 127.0.0.1:9092``: Connect to the SuperLink's Fleet API at the address
+         | ``127.0.0.1:9092``.
+       * | ``--node-config "partition-id=0 num-partitions=2"``: Set the partition ID to ``0`` and the
+         | number of partitions to ``2`` for the SuperNode configuration.
+       * | ``--supernode-address 127.0.0.1:9094``: Set the address and port number where the
+         | SuperNode is listening to communicate with the ClientApp.
+       * | ``--isolation subprocess``: Tells the SuperNode to run the ClientApp in a subprocess.
+
+2. **Terminal 2** Start the second SuperNode:
+
+   .. code-block:: shell
+
+       $ flower-supernode \
+            --insecure \
+            --superlink 127.0.0.1:9092 \
+            --node-config "partition-id=1 num-partitions=2" \
+            --supernode-address 127.0.0.1:9095 \
+            --isolation subprocess
+
```

Comment:
```suggestion
            --isolation subprocess

  .. dropdown:: Understand the command

    * ``--supernode-address 127.0.0.1:9094``: Note how we indicate a different port. This is because if you run two `SuperNodes` on the same machine, they can't make use of the same port to communicate with the sub-process executing the `ClientApp`.


```

Reply:
another option would be to skip showing this second `flower-supernode` command and say: "if you run in a different machine, run exactly as before. If on the same machine, just change the port". This sounds like a better way to me but then the question is: should we present the `--superlink` IPs in a more general way instead of assuming them being `localhost`?

Reply:
What about using a [tab component](https://sphinx-design.readthedocs.io/en/latest/tabs.html)  and show both versions (localhost, different host)?